python/cuml/test/test_pca.py

```diff
@@ -54,17 +56,21 @@ def test_pca_fit(datatype, input_type):
             cuml_res = cuml_res.as_matrix()
         skl_res = getattr(skpca, attr)
         assert array_equal(cuml_res, skl_res, 1e-3, with_sign=with_sign)
+    if stream is not None:
```

Comment:
Don't we need to sync in any case? Also is the sync not needed before line 58?

Reply:
I have it on my list and it's just waiting for PR #475 to be merged as I mentioned [here](https://github.com/rapidsai/cuml/pull/482#issuecomment-484048081) :)

Reply:
Done.